packages/client-sdk-web/src/internal/pubsub-client.ts

```diff
@@ -1,9 +1,7 @@
 import * as pubsub from '@gomomento/generated-types-webtext/dist/CachepubsubServiceClientPb';
```

Comment:
not related to this pr but we should probably rename this client purge pubsub and just call this topic.

Reply:
it's internal, so not public API, but sure we could do that